sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java

```diff
@@ -1688,7 +1688,8 @@ public PollerFlux<BinaryData, BinaryData> beginBeginAzureBatchImageGenerationAsy
                                 .setContext(
                                         requestOptions != null && requestOptions.getContext() != null
                                                 ? requestOptions.getContext()
-                                                : Context.NONE)),
+                                                : Context.NONE)
+                                .setServiceVersion(this.getServiceVersion().getVersion())),
```

Comment:
Set service version query param for polling strategies.

Related PRs: 
azure-core: https://github.com/Azure/azure-sdk-for-java/pull/34216
codegen: https://github.com/Azure/autorest.java/pull/2349